tests/post_training/test_templates/test_ptq_params.py

```diff
@@ -47,6 +47,11 @@
 from tests.common.quantization.mock_graphs import get_nncf_graph_from_mock_nx_graph
 
 
+class MockdedDataset:
```

Comment:
```suggestion
class MockedDataset:
```
Did you mean `Mocked`?

Reply:
done